src/hotspot/share/gc/shared/genCollectedHeap.cpp

```diff
@@ -853,6 +831,15 @@ void GenCollectedHeap::collect(GCCause::Cause cause, GenerationType max_generati
     return;
   }
 
+  bool should_run_young_gc =  (cause == GCCause::_wb_young_gc)
+                           || (cause == GCCause::_gc_locker)
+                DEBUG_ONLY(|| (cause == GCCause::_scavenge_alot))
+                           ;
```

Comment:
I would prefer the semi-colon on the same line as the DEBUG_ONLY, but that's just my preference.
```suggestion
                DEBUG_ONLY(|| (cause == GCCause::_scavenge_alot));
```